.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 public
 npm-debug.log
+package-lock.json
```

Comment:
A lot of dependency issues that we've experienced in a ton of apps for service ops.  e.g.
https://github.com/npm/npm/issues/16866

I can remove this because I don't think this change needs to be discussed as part of this PR